src/peer_review/views/peer_review_view.py

```diff
@@ -1,10 +1,352 @@
 from rest_framework.viewsets import ModelViewSet
+from rest_framework.decorators import action
 from rest_framework.permissions import (
     IsAuthenticated
 )
+from peer_review.permissions import (
+    IsAllowedToCreateDecision,
+)
+from utils.http import DELETE, POST, PATCH, PUT, GET
+from peer_review.models import (
+    PeerReviewDecision,
+    PeerReview,
+)
+from peer_review.serializers import (
+    PeerReviewDecisionSerializer,
+    PeerReviewSerializer,
+)
+from rest_framework.response import Response
+from reputation.models import Contribution
+from django.contrib.contenttypes.models import ContentType
+from discussion.models import Thread
+from reputation.serializers import DynamicContributionSerializer
 
 
 class PeerReviewViewSet(ModelViewSet):
     permission_classes = [
         IsAuthenticated,
     ]
+    serializer_class = PeerReviewSerializer
+    queryset = PeerReview.objects.all()
+
+    def get_serializer_class(self):
+        if self.action == 'create_decision':
+            return PeerReviewDecisionSerializer
+        else:
+            return self.serializer_class
+
+    def _get_contribution_context(self):
```

Comment:
This was burrowed from `user_views.contributions` - it will likely change a bunch before go live